FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/ApiRepository.kt

```diff
@@ -141,12 +141,12 @@ interface ApiRepository : BaseApiRepository {
    * Grab a reply token before uploading a password protected message
    *
    * @param context Interface to global information about an application environment.
-   * @param domain A company domain.
-   * @param domain OIDC token.
+   * @param baseFesUrlPath A base FES URL path.
+   * @param idToken OIDC token.
    */
   suspend fun getReplyTokenForPasswordProtectedMsg(
     context: Context,
-    domain: String,
+    baseFesUrlPath: String,
     idToken: String
   ): Result<MessageReplyTokenResponse>
```

Comment:
`uploadPasswordProtectedMsgToWebPortal` was not modified, surely that doesn't work for a domain that doesn't run their own FES

Reply:
Earlier, the other method was not modified, which meant that:
 - either the modification has no effect
 - or that it would be accidentally calling shared-tenant-fes when there is a custom fes url
 - or that it would attempt (and fail) to call a non-existent custom fes url when there is only shared-tenant-fes

Now it looks ok